packages/common/src/hooks/useFilterBy/useFilterBy.tsx

```diff
@@ -59,7 +63,9 @@ export const useFilterBy = (initialFilterBy?: FilterBy | null): FilterState => {
   };
 
   const onClearFilterRule = (key: string) => {
-    setFilterBy({ ...filterBy, [key]: null });
+    const newFilter = { ...filterBy };
+    delete newFilter[key];
+    setFilterBy(newFilter);
```

Comment:
This is not strictly necessary for this fix. However, the filter plugin I'm used to using for Postgraphile was *extremely* fussy and would throw its toys when you passed it null values. So just feels safer to explicitly delete the key rather than set it to `null`.

Reply:
Nice!